dallinger/experiment_server/dashboard.py

```diff
@@ -593,6 +608,8 @@ def monitoring():
         net_roles=net_roles,
         net_ids=net_ids,
         vis_options=json.dumps(vis_options),
+        configuration=config,
```

Comment:
Maybe call this `config` instead of `configuration`? But I can't see where this is used in the page, maybe you can skip it entirely?

Reply:
No, it cannot be named `config` because it's already used. On other dashboard pages, e.g. `/index` it's also called `configuration`.

Reply:
OK fair enough! I had a look at the location where this is used, dashboard_home.html, and actually this displays the config already. I wonder whether this pre-existing logic should just be retained, but migrated to a dashboard tab so we don't miss it, and adding your new autorecruit checkbox? What do you think?

Reply:
Good catch! I moved it into home now.